website/identifiers/utils.py

```diff
@@ -142,16 +145,16 @@ def get_or_create_identifiers(target_object):
 
         resp = response_dict['response']
         exists = response_dict['already_exists']
-
+        new = response_dict['new']
```

Comment:
same as above, can you just use the `exists` parameter here instead of using `new`?